superset/assets/javascripts/SqlLab/components/AceEditorWrapper.jsx

```diff
@@ -78,13 +78,15 @@ class AceEditorWrapper extends React.PureComponent {
     words = words.concat(Object.keys(columns).map(col => (
       { name: col, value: col, score: 50, meta: 'column' }
     )));
-    this.setState({ words });
-    const completer = {
-      getCompletions: this.getCompletions.bind(this),
-    };
-    if (langTools) {
-      langTools.setCompleters([completer, langTools.keyWordCompleter]);
-    }
+
+    this.setState({ words }, () => {
+      const completer = {
```

Comment:
can we move `const completer = {...}` inside the if block? since we don't need to call `this.getCompletions()` unless `langTools` is truthy.